internal/core/testing.go

```diff
@@ -146,6 +146,7 @@ func getTestClient(t *testing.T, testConfig *TestConfig) (client *scw.Client, cl
 		scw.WithEnv(),
 		scw.WithUserAgent("cli-e2e-test"),
 		scw.WithDefaultOrganizationID("11111111-1111-1111-1111-111111111111"),
+		scw.WithAuth("SCWXXXXXXXXXXXXXXXXX", "11111111-1111-1111-1111-111111111111"),
```

Comment:
This is a little more complicated as:
  - When we are recording cassette we want real tocken
  - When we are not we should use fake one.

But event this will not work as if golden are generated with cassette the access key will not be the same as without cassette

Maybe a simpler solution in your case is to not use the getTestClient but create your own.